examples/fraud-analytics/app/Containerfile

```diff
@@ -0,0 +1,87 @@
+FROM registry.access.redhat.com/ubi9/ubi:9.3 as tf-builder
```

Comment:
can we use UBI 9.6 image? i am sticking to 9.6 for fruad_detection container image.